### PR TITLE
GUNDI-3145: Activity logs for Webhooks

### PR DESCRIPTION
**Pull request by marianobrc**
### What does this PR do?
- Extends activity logs to cover webhook integrations
- Adds test coverage
- Adds some usage examples in the README

### Relevant link(s)
[GUNDI-3145](https://allenai.atlassian.net/browse/GUNDI-3145)

[GUNDI-3145]: https://allenai.atlassian.net/browse/GUNDI-3145?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by davidovichmarcos — APPROVED** [review]
lgtm!

File: app/services/activity_logger.py [hunk 1]
```diff
@@ -82,6 +96,34 @@ async def log_activity(integration_id: str, action_id: str, title: str, level="I
...
+        :param data: Any extra data to be logged as a dict
+        :return: None
+        """
+    logger.debug(f"Logging custom activity: {title}. Integration: {integration_id}. Webhook: {webhook_id}.")
+    await publish_event(
+        event=IntegrationWebhookCustomLog(
+            payload=CustomWebhookLog(
+                integration_id=integration_id,
+                webhook_id=webhook_id,
+                config_data=config_data or {},
```
Comment (chrisdoehring) [inline app/services/activity_logger.py:117]: @marianobrc is there any chance that `config_data` has sensitive values?

**Review by chrisdoehring — COMMENTED** [review]
Thanks @marianobrc
It looks good. I left one comment re: config_data and whether it might include sensitive values.

File: app/services/activity_logger.py (hunk 1 above)
Comment (marianobrc) [reply app/services/activity_logger.py:117]: do you think it's better to remove this cofig_data from logs in both webhooks and actions for now @chrisdoehring?

Comment (chrisdoehring) [reply app/services/activity_logger.py:117]: Ah of course. That makes sense and I agree it's good to leave the config_data in the logs. _Apologies for the slow reply._

**Review by chrisdoehring — APPROVED** [review]

**Pull request merged**